notebook/tests/selenium/utils.py

```diff
@@ -258,3 +272,9 @@ def ctrl(browser, k):
     """Send key combination Ctrl+(k)"""
     ActionChains(browser)\
         .key_down(Keys.CONTROL).send_keys(k).key_up(Keys.CONTROL).perform()
+
+def esc(browser, k):
+    """Send key combination esc+(k)"""
+    ActionChains(browser)\
+        .key_down(Keys.ESCAPE).send_keys(k).key_up(Keys.ESCAPE).perform()
```

Comment:
This isn't necessary - `Esc` isn't used as a modifier key that you have to hold down while pressing other keys. Just call `self.to_command_mode()` and then send the other keys as normal.

Reply:
I see okay. Makes sense, will call self.to_command_mode()